src/runtime_src/hip/core/event.h

```diff
@@ -59,11 +59,22 @@ class command
 
   virtual bool submit() = 0;
   virtual bool wait() = 0;
-  state get_state() const { return cstate; }
-  std::chrono::time_point<std::chrono::system_clock> get_time() { return ctime; }
-  void set_state(state newstate) { cstate = newstate; };
-  type
-  get_type() const { return ctype; }
+  
+  state
+  get_state() const 
+  { return cstate; }
```

Comment:
Please move implementation to new line. Fix all the functions
example:
state 
get_state() const
{
  return cstate;
}

Reply:
Thanks Vamshi, I modified the code.